learning_resources/views.py

```diff
@@ -376,6 +378,125 @@ class ResourceListItemsViewSet(NestedViewSetMixin, viewsets.ReadOnlyModelViewSet
     ordering = ["position", "-child__last_modified"]
 
 
+@extend_schema(
+    parameters=[
```

Comment:
There are 3 parameters here... 2 are relevant for one endpoint, 2 are relevant for the other.

I would try moving the annotation to the methods themselves.

I also *think* you can skip the annotation for the path parameter. I think DRF-spectacular will pick that up automatically.

Reply:
You're right in that you can usually omit the `id` path param, but in this case `drf-spectacular` mislabels it so I'm just specifying it to fix that:

"A unique integer value identifying this learning resource relationship." vs "id of the learning resource"